code/go/0chain.net/blobber/zcn.go

```diff
@@ -69,6 +69,8 @@ func setupOnChain() {
 
 		go keepAliveOnChain()
 
+		go collectDBStats()
```

Comment:
it is better to move `collectDBStats` to `setupDatabase` on datastore.go.  zcn.go is designed for blockchain stuff.

Reply:
got it.